mdtraj/geometry/rdf.py

```diff
@@ -0,0 +1,96 @@
+##############################################################################
+# MDTraj: A Python Library for Loading, Saving, and Manipulating
+#         Molecular Dynamics Trajectories.
+# Copyright 2012-2013 Stanford University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors:
+#
+# MDTraj is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with MDTraj. If not, see <http://www.gnu.org/licenses/>.
+##############################################################################
+
+from __future__ import print_function, division
+
+import itertools
+import re
+
+import numpy as np
+
+from mdtraj.utils import ensure_type
+from mdtraj.geometry.distance import compute_distances
+
+
+__all__ = ['compute_rdf']
+
+
+def compute_rdf(traj, pair_names=None, r_range=None, bin_width=0.005,
+                periodic=True, opt=True):
+    """Compute radial distribution functions for pairs in every frame.
+
+    Parameters
+    ----------
+    traj : Trajectory
+    pair_names : array-like, shape=(2,), optional, default=None
+        Pair of atom names to consider. Atom names are matched using the same
+        regex matching employed by MDTraj's  atom selection DSL.
+    r_range : array-like, shape=(2,), optional, default=(0.0, 1.0)
+        Minimum and maximum radii.
+    bin_width : int, optional, default=0.005
+        Width of the bins in nanometers.
+    opt : bool, default=True
+        Use an optimized native library to calculate the RDF.
+
+    Returns
+    -------
+    r : np.ndarray, shape=(np.diff(r_range) / bin_width - 1), dtype=float
+        Radii values corresponding to bins.
```

Comment:
From the code, the returned `r`s are the center of the bins, but either the left and right bin edges also be reasonable things the user might expect. Maybe the docstring should say that they're the center points.